File: src/mas/hardwareAgent/src/hardwareAgent/TimeData.java
package hardwareAgent;

 /**
 * Author: Thierry Gerritse
 * Class: TimeData.java 
 **/

public class TimeData {
	/**
	 * @var long duration
	 * the length of duration 
	 */
	public long duration;
	
	/**
	 * 
	 * @param duration
	 */
	public TimeData(long duration) {
		this.duration = duration;
	}
	/**
	 * 
	 * @return duration
	 */
	public long getDuration(){
		
		return this.duration;
		
	}
	/**
	 * 
	 * @param duration
	 */
	public void setDuration(long duration){
		
		this.duration = duration;
		
	}
}

File: src/cpp/libraries/rexos_datatypes/src/TimeData.cpp
/* 
 * File:   TimeData.cpp
 * Author: alexander-ubuntu
 * 
 * Created on June 16, 2013, 3:36 PM
 */

#include "rexos_datatypes/TimeData.h"
namespace rexos_datatypes{
    TimeData::TimeData() {
    }

    int TimeData::getDuration(){
        return this->duration;
    }
    void TimeData::setDuration(int duration){
        this->duration = duration;
    }
}